diff -x cc2a776f -Naur ../hitch-adca8d96fa16de1f2e66717ec2b7c95b772a1204.orig/src/tests/test07-nomatch-abort ./src/tests/test07-nomatch-abort
--- ../hitch-adca8d96fa16de1f2e66717ec2b7c95b772a1204.orig/src/tests/test07-nomatch-abort	2015-08-17 11:46:16.000000000 +0200
+++ ./src/tests/test07-nomatch-abort	2015-08-17 13:44:50.630148835 +0200
@@ -30,5 +30,4 @@
 grep -q -c "unrecognized name" $DUMPFILE
 test "$?" = "0" || die "Expected 'unrecognized name' error."
 
-CURL_EXTRA="--resolve site1.example.com:$LISTENPORT:127.0.0.1"
-runcurl site1.example.com $LISTENPORT
+(echo -e 'GET / HTTP/1.0\nHost: site1.example.com\n'; sleep 1) | openssl s_client -connect $LISTENADDR:$LISTENPORT > $DUMPFILE 2>&1
