diff -Naur ../hitch-00b264b5537986fecfa1013cc27ad3b7b771a646.orig/hitch.service ./hitch.service
--- ../hitch-00b264b5537986fecfa1013cc27ad3b7b771a646.orig/hitch.service	1970-01-01 01:00:00.000000000 +0100
+++ ./hitch.service	2015-06-25 14:23:07.120277818 +0200
@@ -0,0 +1,27 @@
+[Unit]
+Description=Network proxy that terminates TLS/SSL connections
+After=syslog.target network.target
+
+[Service]
+
+#
+# If you want to make changes to this file, please copy it to 
+# /etc/systemd/system/hitch.service and make your changes there.
+# This will override the file kept at /lib/systemd/system/hitch.service
+#
+# Configuration may be done in /etc/hitch/hitch.params
+#
+
+PIDFile=/run/hitch/hitch.pid
+
+# Maximum size of the corefile. 
+LimitCORE=infinity
+
+Type=simple
+PrivateTmp=true
+ExecStart=/usr/sbin/hitch --pidfile=/run/hitch/hitch.pid --config=/etc/hitch/hitch.conf
+
+
+[Install]
+WantedBy=multi-user.target
+
diff -Naur ../hitch-00b264b5537986fecfa1013cc27ad3b7b771a646.orig/hitch.tmpfilesd.conf ./hitch.tmpfilesd.conf
--- ../hitch-00b264b5537986fecfa1013cc27ad3b7b771a646.orig/hitch.tmpfilesd.conf	1970-01-01 01:00:00.000000000 +0100
+++ ./hitch.tmpfilesd.conf	2015-06-25 14:44:15.785888764 +0200
@@ -0,0 +1 @@
+d /run/hitch 0755 hitch hitch -
