diff -ru src/tests/configs.orig/test08a.cfg src/tests/configs/test08a.cfg
--- src/tests/configs.orig/test08a.cfg	2015-07-23 15:22:17.000000000 +0200
+++ src/tests/configs/test08a.cfg	2015-10-15 21:29:42.565891140 +0200
@@ -8,7 +8,7 @@
 keepalive = 3600
 chroot = ""
 user = "nobody"
-group = "nogroup"
+group = "nobody"
 quiet = on
 syslog = on
 quiet = on
diff -ru src/tests/configs.orig/test08b.cfg src/tests/configs/test08b.cfg
--- src/tests/configs.orig/test08b.cfg	2015-07-23 15:22:17.000000000 +0200
+++ src/tests/configs/test08b.cfg	2015-10-15 21:29:42.566891197 +0200
@@ -8,7 +8,7 @@
 keepalive = 3600
 chroot = ""
 user = "nobody"
-group = "nogroup"
+group = "nobody"
 quiet = on
 syslog = on
 syslog-facility = "info"
diff -ru src/tests/configs.orig/test08c.cfg src/tests/configs/test08c.cfg
--- src/tests/configs.orig/test08c.cfg	2015-07-23 15:22:17.000000000 +0200
+++ src/tests/configs/test08c.cfg	2015-10-15 21:29:42.566891197 +0200
@@ -8,7 +8,7 @@
 keepalive = 3600
 chroot = ""
 user = "nobody"
-group = "nogroup"
+group = "nobody"
 quiet = on
 syslog = on
 syslog-facility = "daemon"
diff -ru src/tests/configs.orig/test08d.cfg src/tests/configs/test08d.cfg
--- src/tests/configs.orig/test08d.cfg	2015-07-23 15:22:17.000000000 +0200
+++ src/tests/configs/test08d.cfg	2015-10-15 21:29:42.566891197 +0200
@@ -9,7 +9,7 @@
 keepalive = 3600
 chroot = ""
 user = "nobody"
-group = "nogroup"
+group = "nobody"
 quiet = on
 syslog = on
 syslog-facility = "daemon"
